packages/patrol_cli/lib/src/features/test/android_test_backend.dart

```diff
@@ -60,74 +92,126 @@ class AndroidAppOptions {
   }
 }
 
-class AndroidTestBackend {
+/// Provides functionality to build, install, run, and uninstall Android apps.
```

Comment:
I don't see install method in this class

Reply:
Installation is performed as part of `AndroidTestBackend.install()`. It calls Gradle task `:app:connectedDebugAndroidTest`, which depends on `:app:installDebug` and `:app:installDebugAndroidTest`.

I was thinking about making adding the `install()` function to the `TestBackend` interface, but I don't see the benefits of doing it.